setup.py

```diff
@@ -493,6 +494,9 @@ def pxd(name):
                     'pxdfiles': ['_libs/src/util', '_libs/lib'],
                     'depends': tseries_depends,
                     'sources': npdt_srces},
+    '_libs.tslibs.conversion': {'pyxfile': '_libs/tslibs/conversion',
+                                'depends': tseries_depends,
+                                'sources': npdt_srces},
```

Comment:
comment from before, rename npdt_srces -> np_datetime_sources.  Please don't abbreviate things, it makes it *much* harder to read.

Reply:
Yah, this change definitely made it into one or two of the ongoing PRs, hasn't caught up to all of them yet.